modules/ROOT/pages/addition/instance-requirements.adoc

```diff
@@ -4,8 +4,9 @@ In order to be managed by NOM a Neo4j instance **must**:
 
 * be running a Neo4j Enterprise Edition v4.4.0 or higher (including all 5.x versions).
 
+For the query log collection feature to work correctly `db.logs.query.enabled` needs to be set to `INFO` or `VERBOSE` in the `neo4j.conf` file.
 
-For the metrics collection feature to work correctlty these configuration values need to be set: 
+For the metrics collection feature to work correctly these configuration values need to be set in the `neo4j.conf` file:
```

Comment:
Even though most DBAs should be familiar with it, should we explain or cross-reference some information about what this file is? Ideally a link to the main Neo4j documentation, but not sure what the best practices are.